docs/config.md

```diff
@@ -908,3 +908,25 @@ how a task or flow is _currently_ configured. The information output by
 these commands change as you make further customizations to your
 project's `cumulusci.yml` file.
 ```
+
+## Loading additional yaml configuration from a file
+
+CumulusCI supports loading in an additional yaml file from the command line with the `--load-yml <path/to/file>` option on `cci task` and `cci flow` commands. This can be useful if you have one-off automation configurations that you want to keep out of the main project's configuration and load only in special cases.
```

Comment:
Thanks for including documentation with your PR. I think that there should be a mention of this option in the configuration scopes [section above](https://github.com/SFDO-Tooling/CumulusCI/blob/a636dd1e0515ae7b3572cee839ec5f30eef2f151/docs/config.md?plain=1#L640).

Reply:
Thanks for the review. I'll get these addressed shortly.